src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -89,41 +128,36 @@ If you want to customize APM agent behavior, you can use our APM agent APIs. Her
-</Callout>
+If you want to automate browser monitoring deployment across multiple web apps, you can use NerdGraph and our browser agent npm package to configure and deploy the agent as part of your application build process.
 
-For possible performance or other reasons, you may wish to delay the initialization of the browser agent from the start of the page load cycle. You won't be able to use the `newrelic` API until the agent is fully initialized. This delay also creates incomplete data in your browser monitoring UI.
+For more on this, see [instrument multiple apps with npm and NerdGraph](/docs/apis/nerdgraph/examples/combining-npm-nerdgraph/).
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
 
-Once the agent is fully initialized, it functions as expected. Any delay also affects when the agent wraps globals or subscribes to events relative to other libraries or code that affects captured data.
-
-You can delay the agent initialization in these ways:
+## What's next? [#whats-next]
 
-* Add a `script-src` tag with the `async` attribute.
-* Add a `script-src` tag with the `defer` attribute.
-* Use dynamic import or chronologically late placement of the loader code.
+When you've installed the browser agent, wait a few minutes and look for the data in [our browser monitoring UI](https://one.newrelic.com/nr1-core?filters=%28domain%3D%27BROWSER%27%20AND%20type%3D%27APPLICATION%27%29&state=1f7bae2d-dd3a-0027-3079-b60bcd19ded9&_gl=1*15f78ub*_ga*MTE0MTA2MTA5MS4xNzE4OTI0MDkw*_ga_R5EF3MCG7B*MTcyMTMxMzk2MC41Ny4xLjE3MjEzMTg2MzcuNDcuMS4xMDYwNTI3Njcw).
 
-Your page's `src` must point to a valid agent loader version in our CDN. For example, the URL that always contains our current Pro+SPA agent release is `https://js-agent.newrelic.com/nr-loader-spa-current.min.js`. [CSP is required](/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring#csp) to load external third-party scripts.
+If it's been more than 15 minutes and you're not seeing data, see [troubleshoot your browser installation](/docs/browser/browser-monitoring/troubleshooting/troubleshoot-your-browser-monitoring-installation/).
 
-To determine when you can start calling the `newrelic` API, subscribe to an event at the global scope:
+Once you're seeing browser data in the UI, here are some next steps on how you can get the most out that data: 
 
-```js
-window.addEventListener('newrelic', (evt) => {
-  if (evt.detail.loaded) {
-    // you can start using newrelic.interaction(), etc. now
-  }
-})
-```
+* View your app data on the [summary page](/docs/browser/browser-monitoring/getting-started/browser-summary-page).
+* Now that you've instrumented with the browser agent, go deeper with your data with our [improve your website tutorial](/docs/journey-performance/guide-to-monitoring-core-web-vitals). 
```

Comment:
```suggestion
* [Monitor your core web vitals](/docs/tutorial-improve-site-performance/guide-to-monitoring-core-web-vitals/)
```

Reply:
Either use this updated core web vitals link, or link to the first page in that tutorial: https://docs.newrelic.com/docs/tutorial-improve-site-performance/improve-website-performance/